src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiService.scala

```diff
@@ -32,6 +33,15 @@ trait BillingApiService extends HttpService with PerRequestCreator with UserInfo
           UserService.props(userServiceConstructor, userInfo),
           UserService.RemoveUserFromBillingProject(RawlsBillingProjectName(projectId), RawlsUserEmail(userEmail)))
       }
+    } ~
+    path("billing") {
```

Comment:
why does this endpoint exist here and under /user/billing? the only difference seems to be that this one has a 7 minute timeout specified